elliottlib/errata.py

```diff
@@ -635,25 +635,17 @@ def add_jira_bugs_with_retry(advisory_id: int, bugids: List[str], noop: bool = F
         if noop:
             logger.info('Dry run: Would have attached bugs')
             continue
-        results = add_multi_jira_issues(advisory_id, chunk_of_bugs)
-        for i, result in enumerate(results):
-            if result.status_code != 201:
-                rt = add_jira_issue(advisory_id, chunk_of_bugs[i])
-                if rt.status_code != 201:
-                    errata_error = rt.json()
```

Comment:
ximin, I'd like to keep the error checking logic we have here

Reply:
I think errata_tool already handled some of the check https://github.com/red-hat-storage/errata-tool/blob/master/errata_tool/connector.py#L201

Reply:
as for the issue of bug already in another advisory, that is the bug of we didn't correctly handle placeholder jira bug, and we now excluded it in bug search, that situation may not happened again